src/__experimental__/components/textbox/textbox.stories.js

```diff
@@ -16,17 +18,33 @@ const defaultStoryPropsConfig = {
   inputWidthEnabled: true
 };
 
-storiesOf('Experimental/Textbox', module)
+const store = new Store(
+  {
+    value: ''
+  }
+);
+
+const setValue = (ev) => {
+  store.set({ value: ev.target.value });
+};
 
-  .add('Basic', () => {
+storiesOf('Experimental/Textbox', module)
+  .add('default', () => {
     return (
       <Textbox
         placeholder={ text('placeholder') }
         { ...getCommonTextboxStoryProps() }
       />
     );
+  }, {
+    info: {
+      text: info,
+      propTables: [OriginalTextbox],
+      propTablesExclude: [State, Textbox],
+      notes: { markdown: notes }
```

Comment:
I think textbox needs to handle special characters
`knobs: {
   escapeHTML: false
}`

Reply:
added